config/default/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
why did you have to create a service account?

Reply:
I would not like to use the default service account